onfleet/models.py

```diff
@@ -281,3 +281,12 @@ def parse(self, obj):
 
         return worker
 
+
+class Container(object):
```

Comment:
After looking at onfleet's api again, and how it is inconsistent for containers.. I don't think this is worth doing at all. It doesn't do anything for us.